src/parse/asp/exec.go

```diff
@@ -96,6 +185,167 @@ func execGetCachedOutput(key execKey, args []string) (output string, found bool)
 	}
 }
 
+// execGitBranch returns the output of a git_branch() command.
+//
+// git_branch() returns the output of `git symbolic-ref -q --short HEAD`
+func execGitBranch(s *scope, args []pyObject) pyObject {
+	short := args[0].IsTruthy()
+
+	cmdIn := make([]pyObject, 3, 5)
+	cmdIn[0] = pyString("git")
+	cmdIn[1] = pyString("symbolic-ref")
+	cmdIn[2] = pyString("-q")
```

Comment:
The `pyString(...)` conversion seems awkward and unnecessary to me. See comment above at `doExec()`

Reply:
I'm still doing this for now that way `doExec()` can take a `pyString()` or a `pyList()` if there are future callers into `doExec()`.  It can be moved to `[]string` without trouble.